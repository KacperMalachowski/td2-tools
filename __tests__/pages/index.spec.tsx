import React from "react";
import renderer from "react-test-renderer";

import Home from '../../src/pages/index';

describe('Home component', () => {
  it("renders correectly", () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})