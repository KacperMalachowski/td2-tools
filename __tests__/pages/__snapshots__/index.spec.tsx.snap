// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correectly 1`] = `
<h1>
  Hello World
</h1>
`;
